airflow/models/taskinstance.py

```diff
@@ -915,6 +916,12 @@ def signal_handler(signum, frame):
                 start_time = time.time()
 
                 self.render_templates(context=context)
+                # Export context to make it available for operators to use.
+                airflow_context_vars = context_to_airflow_vars(context, in_env_var_format=True)
+                self.log.info("Exporting the following env vars:\n%s",
+                              '\n'.join(["{}={}".format(k, v)
```

Comment:
```suggestion
                vars = [list(i) for i in airflov_context_vars.itmes()] 
                self.log.info("Exporting the following env vars:\n" + "%s=%s\n" * len(vars), *sum(vars, []))
```
What do you think? In this way we will fix one pylint error (not caught yet because this file is still in todo) :)